### PR TITLE
OHRI-507 Feature/ohri 507 customisable action form title

### PR DESCRIPTION
**Pull request by larslemos**
This PR covers below scope:

- [ ] Refactor Encounter Component to Allow Overflow Custom Title
- [ ] Refactor Custom Actions
- [ ] Add  Custom Launcher 
- [ ] Render Title mode on Form

### CODE REVIEW

File: src/components/encounter-list/encounter-list.component.tsx [hunk 1]
```diff
@@ -177,21 +182,21 @@ const EncounterList: React.FC<EncounterListProps> = ({
...
                   e.preventDefault();
                   if (actionItem.mode == 'edit') {
-                    launchFormInEditMode(
+                    launchEncounterForm(
                       applyFormIntent(actionItem.intent, getForm(actionItem.form.package, actionItem.form.name)),
                       actionItem.encounterUuid,
-                      forceComponentUpdate,
                     );
                   } else if (actionItem.mode == 'enter') {
                     launchForm(
```
Comment (pirupius) [inline src/components/encounter-list/encounter-list.component.tsx:190]: @larslemos don't we need to edit this as well?

File: src/components/encounter-list/encounter-list.component.tsx [hunk 2]
```diff
@@ -177,21 +182,21 @@ const EncounterList: React.FC<EncounterListProps> = ({
                 onClick={e => {
                   e.preventDefault();
                   if (actionItem.mode == 'edit') {
-                    launchFormInEditMode(
+                    launchEncounterForm(
```
Comment (pirupius) [inline src/components/encounter-list/encounter-list.component.tsx:185]: This doesn't seem to be passing/accepting a mode compared to the `launchFormInEditMode` won't this bring issues?

File: src/components/encounter-list/encounter-list.component.tsx (hunk 1 above)
Comment (larslemos) [reply src/components/encounter-list/encounter-list.component.tsx:190]: Thank you for raising this, I was making use of the additional hook, existing at parent level.

File: src/components/encounter-list/encounter-list.component.tsx (hunk 2 above)
Comment (larslemos) [reply src/components/encounter-list/encounter-list.component.tsx:185]: Yes, this missed me. Updated

**Review by pirupius — APPROVED** [review]

**Pull request merged**